helix-rest/src/test/java/org/apache/helix/rest/server/TestClusterAccessor.java

```diff
@@ -243,6 +243,7 @@ public void testAddConfigFields() throws IOException {
     oldConfig.getRecord().update(configDelta.getRecord());
     Assert.assertEquals(newConfig, oldConfig,
         "cluster config from response: " + newConfig + " vs cluster config actually: " + oldConfig);
+    System.out.println("new config:" + newConfig.toString());
```

Comment:
Why printout again if it is equal?

Reply:
+1

I see system out println quite a lot in your other PRs. If there is a specific reason, please state. Otherwise, please avoid it. Otherwise, it will distract the reviewers' attention and slow down the review process unnecessarily.

Reply:
Is this PR for review? Looks like it's only for debugging purpose instead of fixing the test failure. @kaisun2000 Maybe you can mark it as WIP if it's only for testing.

Reply:
This is to figure out what goes wrong, as this test:

1/ does not fail in my local run

2/ fails some times in git hub run.

3/ start to fail recently.

Now the root cause is clear. Basically, the INSTANCE_CAPACITY_KEYS list field can appear in the test and fail the test as the test does not expect this field to "magically" appear, depends on timing. 

It turned out to be the case that Waged Rebalancer would add this field to the cluster config when they finished the first global run.  Depending on the global run timing, the can fail. 

See the detailed analysis in the following issue: https://github.com/apache/helix/issues/1336

Reply:
Please keep the debug log in your local environment.
Your debug logs have been reviewed by at least 3 engineers. And I consider this as an unnecessary overhead : (